packages/host/app/components/ai-assistant/llm-select.gts

```diff
@@ -42,6 +42,8 @@ const LLMSelect: TemplateOnlyComponent<Signature> = <template>
       border: none;
       padding: 0;
       width: auto;
+      min-width: 80px;
+      height: 43.5px;
```

Comment:
Is this calculated from something?

Reply:
It's calculated from the item pill height